File: TwoCaptcha/Captcha/Lemin.cs
namespace TwoCaptcha.Captcha
{
    public class Lemin : Captcha
    {
        public Lemin() : base()
        {
            parameters["method"] = "lemin";
        }

        public void SetCaptchaId(string captchaid)
        {
            parameters["captcha_id"] = captchaid;
        }

        public void SetUrl(string url)
        {
            parameters["pageurl"] = url;
        }

        public void SetApiServer(string apiServer)
        {
            parameters["api_server"] = apiServer;
        }
    }
}

File: TwoCaptcha/Captcha/Lemin.java
package com.twocaptcha.captcha;

public class Lemin extends Captcha {

    public Lemin() {
        super();
        params.put("method", "lemin");
    }

    public void setApiServer(String apiServer) {
        params.put("api_server", apiServer);
    }

    public void setСaptchaId(String captchaId) {
        params.put("captcha_id", captchaId);
    }    

    public void setUrl(String url) {
        params.put("pageurl", url);
    }

}
